modules/user_accounts/php/edit_user.class.inc

```diff
@@ -327,68 +342,68 @@ class Edit_User extends \NDB_Form
                 //get existing sites for examiner
                 $prev_sites = $DB->pselect(
                     "SELECT centerID
-             FROM examiners_psc_rel epr
-             WHERE examinerID=:eid",
+                            FROM examiners_psc_rel epr
+                            WHERE examinerID=:eid",
                     array("eid" => $examinerID)
                 );
 
                 //get sites where user is already an examiner at for compare
                 foreach ($prev_sites as $row => $center) {
                     array_push($ex_prev_sites, $center['centerID']);
                 }
-            }
-
-            foreach ($ex_curr_sites as $k => $v) {
 
-                //Check if examiner already in db for site
-                $result = $DB->pselectRow(
-                    "SELECT epr.centerID
-                     FROM examiners_psc_rel epr 
-                     WHERE epr.examinerID=:eid AND epr.centerID=:cid",
-                    array(
-                     "eid" => $examinerID,
-                     "cid" => $v,
-                    )
-                );
+                foreach ($ex_curr_sites as $k => $v) {
```

Comment:
you're missing a `}` before this line (and subsequently have an extra at the end), otherwise new examiners won't have any site affiliation. Previously this code was run regardless of the empty status of $examinerID, and now it is only run when it is set. Setting the examiner status of an existing user for the first time breaks the page as a result.

Reply:
This one is the one that sketches me out the most. I know that it does break when this condition is hit, but I didn't check if moving the `}` would fix it. Give it a shot and lmk @cmadjar.